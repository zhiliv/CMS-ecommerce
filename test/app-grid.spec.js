import { shallowMount } from '@vue/test-utils'
import Row from '@/components/app/grid/grid.vue'

describe('app-grid.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Row, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple',
        styles: 'background: white',
        id: 'indent'
      },
    })
  })
  it('Содержимое html для app-grid ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
