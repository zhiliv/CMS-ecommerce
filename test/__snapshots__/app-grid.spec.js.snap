// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-grid.js Содержимое html для app-grid  1`] = `
<div
  class="grid simple"
  id="indent"
  style="background: white;"
/>
`;
